mbed_lstools/lstools_win7.py

```diff
@@ -134,6 +142,19 @@ def get_connected_mbeds(self):
         """
         return [m for m in self.get_mbeds() if os.path.exists(m[0])]
 
+    def get_connected_mbeds_usb_ids(self):
+        """! Function  return mbeds with existing mount point's
+             target ID mapped to their target USB ID
+        @return Returns {<target_id>: <target_usb_id>, ...}
+        @details Helper function
+        """
+        connected_mbeds_ids = {}
+        for mbed in self.get_connected_mbeds():
+            htm_target_id = self.get_mbed_htm_target_id(mbed[0])
+            target_id = htm_target_id if htm_target_id else mbed[1]
```

Comment:
It looks like the logic here is duplicated from the function `discover_connected_mbeds`. Do you think you could break out this logic into a separate function and use it in both functions?